airbyte-integrations/connectors/source-surveymonkey/integration_tests/expected_records.txt

```diff
@@ -0,0 +1,427 @@
+{"stream":"surveys","data":{"title":"jjj","nickname":"","language":"en","folder_id":"0","category":"","question_count":0,"page_count":1,"response_count":0,"date_created":"2023-01-17T09:17:00","date_modified":"2023-01-17T09:17:00","id":"510388524","buttons_text":{"next_button":"Next","prev_button":"Prev","done_button":"Done","exit_button":""},"is_owner":true,"footer":true,"theme_id":"10292568","custom_variables":{},"href":"https://api.surveymonkey.com/v3/surveys/510388524","analyze_url":"https://www.surveymonkey.com/analyze/VXMmVNBbmOp9KTSvXdhjIr3FHnqleAX32lr9MLBIOE0_3D","edit_url":"https://www.surveymonkey.com/create/?sm=VXMmVNBbmOp9KTSvXdhjIr3FHnqleAX32lr9MLBIOE0_3D","collect_url":"https://www.surveymonkey.com/collect/list?sm=VXMmVNBbmOp9KTSvXdhjIr3FHnqleAX32lr9MLBIOE0_3D","summary_url":"https://www.surveymonkey.com/summary/VXMmVNBbmOp9KTSvXdhjIr3FHnqleAX32lr9MLBIOE0_3D","preview":"https://www.surveymonkey.com/r/Preview/?sm=TPlNncbuCs17cvxwjuS74VC03_2FOcqpP_2F03m2gerTSI_2FQvLWoY2yn_2FWxLDmxYOp5L"},"emitted_at":1674149658247}
+{"stream":"survey_responses","data":{"id":"12706126725","recipient_id":"","collection_mode":"default","response_status":"completed","custom_value":"","first_name":"","last_name":"","email_address":"","ip_address":"124.123.178.184","logic_path":{},"metadata":{"contact":{}},"page_path":[],"collector_id":"405437100","survey_id":"306079584","custom_variables":{},"edit_url":"https://www.surveymonkey.com/r/?sm=hNu3QJYf07WiUPOwxCYcARURFGB3ruOrs9slcTHVhmhgDhoNJ0k7w3jCvo0nLM40","analyze_url":"https://www.surveymonkey.com/analyze/browse/5jPPwKLnlqevaUQom_2BgYAWJWlrKNA2ZFTOYrMBrqW2c_3D?respondent_id=12706126725","total_time":62,"date_modified":"2021-06-01T17:40:54+00:00","date_created":"2021-06-01T17:39:51+00:00","href":"https://api.surveymonkey.com/v3/surveys/306079584/responses/12706126725","pages":[{"id":"165250506","questions":[{"id":"652286715","answers":[{"choice_id":"4285525064"}]},{"id":"652286721","answers":[{"choice_id":"4285525084"}]},{"id":"652286716","answers":[{"choice_id":"4285525070"}]},{"id":"652286718","answers":[{"choice_id":"4285525079"}]},{"id":"652286722","answers":[{"choice_id":"4285525089"}]},{"id":"652286717","answers":[{"choice_id":"4285525074"}]},{"id":"652286723","answers":[{"choice_id":"4285525095"}]},{"id":"652286714","answers":[{"choice_id":"4285525058","row_id":"4285525061","choice_metadata":{"weight":"0"}}]}]}]},"emitted_at":1674149660097}
```

Comment:
Future note: We don't have any PII in here at the moment, but we could if the person taking this fake survey provides an email address.

Reply:
Basically, all these surveys were created 2 years ago and stay without changes since that time.

> I think, because the IP addresses here have no other related data (email, name, etc) this is probably OK?
> 
> @YowanR I think the final call should come from you. Assigning you as the reviewer

Basically, all these surveys were created 2 years ago and stay without changes since that time.

as for IPs: I suppose they belong to internet provers and usually dynamically change. So as for me, they should not be a problem as well.